tests/metrics/genai/test_genai_metrics.py

```diff
@@ -830,8 +830,7 @@ def test_answer_relevance_metric():
             "references and to\nunderstand them before completing the task.\n"
             f"\nInput:\n{input}\n"
             f"\nOutput:\n{mlflow_prediction}\n"
-            "\nAdditional information used by the model:\nkey: context\nvalue:\n"
-            f"{mlflow_ground_truth}\n"
+            "\nAdditional information used by the model:\n\n"
```

Comment:
I think we shouldn't be seeing this as well since there is no more additional info